Makefile

```diff
@@ -0,0 +1,31 @@
+GOOS=$(shell go env GOOS)
```

Comment:
As opposed to deleting and re-adding as a new file, it would be better if you could do a [`git mv`](https://git-scm.com/docs/git-mv) to rename the filename.

Reply:
I did do a `git mv` originally, but git decides automatically whether a file has been deleted or renamed based on how similar the content is.  What I could do instead is one commit with the name change, and then another with the line changes, that should make the history more clear.